Dependencies/xr/Source/ARCore/Include/IXrContextARCore.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include <arcore_c_api.h>
+
+typedef struct IXrContextARCore
+{
+    virtual bool IsInitialized() const = 0;
+    virtual ArSession* XrSession() const = 0;

```

Comment:
Should this just be named Session? The xr prefix seems redundant.

Reply:
the prefix was to prevent collisions with the underlying Session variable. I don't have a preference here. if theres a good way to have functions and fields have the same name in a struct we could go that route instead.